### DISCUSSION TITLE
How to access the dashboard?

### DISCUSSION THREAD

**Discussion by coding-beginner-1** [hjlarry/flask-shop#99] [Q&A]
Hello there,
I'm new to Python and this flask app works very well on my server. However I don't know how to access the dashboard. Just inserting "/dashboard" doesn't seem to work and I am out of ideas. I would welcome any help. :)
Johannes
![image](https://user-images.githubusercontent.com/56784675/217510373-9ffa87c2-7f70-452a-b0ad-0dafef0f3e10.png)

**Comment by dedalgr** [comment]
Try to login as admin and use link

На ср, 8.02.2023 г., 12:26 ч. johannesaschmid ***@***.***>
написа:
